### PR TITLE
add Django 5.0 support

### PR DESCRIPTION
**Pull request by stefano-esubasta**
upon trying to test this project on Django 5.0, only `test_validation_filters` failed;
after fixing it, everything seems to be working.
hope this helps!

### CODE REVIEW

**Comment by WhyNotHugo** [comment]
Looks like this fails on Python 3.9: https://github.com/WhyNotHugo/django-afip/actions/runs/8942289677/job/24640657514?pr=209#step:7:228

File: tests/test_admin.py [hunk 1]
```diff
@@ -190,46 +190,45 @@ def test_validation_filters(admin_client: Client) -> None:
         receipt=failed_validation_receipt,
     )
 
+    html = (
+        '<input class="action-select" name="_selected_action" value="{}" '
+        'type="checkbox" aria-label="Select this object for an action - {}">'
```
Comment (WhyNotHugo) [inline tests/test_admin.py:195]: I think that this should work on Django 5.1 and older versions too:

```suggestion
        'type="checkbox"
```

Comment (stefano-esubasta) [reply tests/test_admin.py:195]: it was failing for me until i checked the response and added that missing attribute :sweat_smile:
maybe there's something wrong with my environment/setup :(

Comment (stefano-esubasta) [reply tests/test_admin.py]: thanks! i tried different solutions and `test_validation_filters` now passes with the changes pushed in commit e1504564f467ce4776300b903dee12436e01052b for the different django versions. now, it seems that `test_html_view` is failing the same way as commit 7272160d7b8122d21f70064b29f56087d429f59e (https://builds.sr.ht/~whynothugo/job/1212081#task-tox-mariadb-79). i'll see if i can figure out a fix and push it as soon as i can :)

**Comment by WhyNotHugo** [comment]
Thanks!

**Pull request merged**